invenio_communities/members/services/service.py

```diff
@@ -250,12 +251,15 @@ def _invite_factory(self, identity, community, role, visible, member,
                 topic=community,
                 expires_at=invite_expires_at(),
                 uow=uow,
-
             )
-            # TODO: create request comment instead of description in case a
+
             # message was provided.
             if message:
-                pass
+                data = {"payload": {"content": message}}
```

Comment:
As discussed IRL, I'm adding this in the members service in order **not to** delegate the responsability of creating the comment to the requests service.

P.S. I've noticed that all factory methods (invite, add, etc.) have a `message` parameter. In the case of invite is used (in this PR), but in the [add](https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/members/services/service.py#L194) it is not. Is it there to have a common func api? or is a remaining of all code and could be removed?